mkt/account/tests/test_views.py

```diff
@@ -326,7 +326,7 @@ def test_installed_pagination(self):
         eq_(data['meta']['limit'], 2)
         prev = urlparse(data['meta']['previous'])
         eq_(next.path, self.list_url)
-        eq_(QueryDict(prev.query).dict(), {u'limit': u'2', u'offset': u'0'})
```

Comment:
Shouldn't be an issue, but it makes me nervous that it changed. Making a note to investigate.